build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group = 'io.pinecone'
-version = '0.6.0' // [pc:VERSION_NEXT]
+version = pineconeClientVersion // [pc:VERSION_NEXT]
```

Comment:
For my own learning: this file has implicit access to fields in `gradle.properties`?

Reply:
Yes!

Reply:
Also I tested it by releasing on local and making sure I was getting correct jars, so the versioning works!